sources/Outputs/src/Par.cpp

```diff
@@ -475,5 +486,51 @@ Par::updatePropParameters(boost::shared_ptr<parameters::ParametersSet> set) {
   set->addReference(helper::buildReference("generator_QRef0Pu", "targetQ_pu", "DOUBLE"));
   set->addReference(helper::buildReference("generator_QPercent", "qMax_pu", "DOUBLE"));
 }
+
+boost::shared_ptr<parameters::ParametersSet>
+Par::writeStaticVarCompensator(const inputs::StaticVarCompensator& svarc) {
+  auto set = boost::shared_ptr<parameters::ParametersSet>(new parameters::ParametersSet(svarc.id));
+
+  set->addMacroParSet(boost::make_shared<parameters::MacroParSet>(macroParameterSetStaticCompensator_));
+
+  double value = computeBPU(svarc.bMax, svarc.VNom);
+  set->addParameter(helper::buildParameter("SVarC_BMaxPu", value));
+
+  value = computeBPU(svarc.bMin, svarc.VNom);
+  set->addParameter(helper::buildParameter("SVarC_BMinPu", value));
+
+  value = computeBPU(svarc.b0, svarc.VNom);
+  set->addParameter(helper::buildParameter("SVarC_BShuntPu", value));
+
+  value = svarc.slope * Sb_ / svarc.VNom;
```

Comment:
maybe you can create a method to evaluate slopePu, just to be similat to computeBPu

Reply:
It is only needed here so I'm not sure it is usefull

Reply:
ok